Merge Development : Moving Changes from Development to Staging Branch

hemalatagaikwad opened the pull request:
## Description

> No Screenshots : 24th October 2024

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

alok-nakhate reviewed (approved):
Changes are reviewed and approved.

DeepashriDatar committed:
Enhancement added to the toast component

Added new properties like
State: 'Basic', 'Info', 'error' & 'success'
Layout : Default, Download, Chat & Request
& other properties like ProgressBarValue & filename
Added stroies for different layouts

DeepashriDatar committed:
Merge branch 'development' into users/deepashri/task-776-enhancements-in-toaster

DeepashriDatar committed:
removed commented code which is not required

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/task-776-enhancements-in-toaster

DeepashriDatar committed:
Removed layout & bordercolor property from UI as it is none of use now.

DeepashriDatar committed:
Merge branch 'users/deepashri/task-776-enhancements-in-toaster' of https://github.com/Wai-Technologies/raaghu-react into users/deepashri/task-776-enhancements-in-toaster

rahulp34 committed:
commited changes

rahulp34 committed:
commited changes

rahulp34 committed:
commited changes

rahulp34 committed:
commited changes

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/task-776-enhancements-in-toaster

prakashmhaske committed:
commit changes for spinner enhancement

DeepashriDatar committed:
Different stories & props are added

Added 3 stories
Added different type of props with respect to stories
& few html & css changes done

DeepashriDatar committed:
added changes for search (child items) removed commented & unwanted code

rahulp34 committed:
commited changes for toast variables that is border-radius, toast padding, fonts, colors

rahulp34 committed:
commited changes

rahulp34 committed:
commit merge

rahulp34 committed:
Merge branch 'users/deepashri/task-776-enhancements-in-toaster' of https://github.com/Wai-Technologies/raaghu-react into users/deepashri/task-776-enhancements-in-toaster

rahulp34 committed:
commited changes

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/task-776-enhancements-in-toaster

DeepashriDatar committed:
Merge branch 'development' into users/deepashri/task-776-enhancements-in-toaster

DeepashriDatar committed:
error resolved

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/task-776-enhancements-in-toaster

rahulp34 committed:
Merge remote-tracking branch 'origin/development' into users/deepashri/task-776-enhancements-in-toaster

DeepashriDatar committed:
removed alert-warning part in common scss

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/task-820-enhancements-in-Side_navigation

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/task-820-enhancements-in-Side_navigation

DeepashriDatar committed:
Fixed issue of mobile views (small & large) footer overlapping issue

DeepashriDatar committed:
changed the class

DeepashriDatar committed:
Added fixes for clear the dropdowns after Save click

abhijeetjawale1 committed:
changes for avtar implementation

rahulp34 committed:
commited changes- removed .icon file from css for toast element

rahulp34 committed:
commited changes- added color variants for icon stroke and icon fill property removed as this property not required as per  figma wifreframe

rahulp34 committed:
commited changes- added state and layout in jest files

rahulp34 committed:
commit merge

rahulp34 committed:
Merge branch 'users/deepashri/task-776-enhancements-in-toaster' of https://github.com/Wai-Technologies/raaghu-react into users/deepashri/task-776-enhancements-in-toaster

poojagurav21 committed:
committed changes for text color in dark theme

poojagurav21 committed:
committed changes for icon color changes

poojagurav21 committed:
Merge branch 'development' into user/pooja/bug-867-collaspible-container-icon-issue

poojagurav21 committed:
committed changes for  Design System > Components > Dynamic Entity Properties> Default> Small, Large mobile overlap Parameter. #870

abhijeetjawale1 committed:
changes done for the avatar as per figma

abhijeetjawale1 committed:
update changes for text

abhijeetjawale1 committed:
Merge branch 'development' into users/abhijeet-task-implement-different-types-of-avtaars

rahulp34 committed:
Rahul- commited changes- icon flickering issue resolved, initially subscribe to text showing in two lines now showing in one line

DeepashriDatar committed:
added gaps between the fields

rahulp34 committed:
commited changes- completed issue for text visibility on background image component, action dropdown issue fixed for bloglist small devices, billing cards alignment issue fix in billing component

abhijeetjawale1 committed:
changes done for the rating enhancement

prakashmhaske committed:
commit chnages for mobile table issue

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/bug-898-component-ClaimType-Default-AfterSubmitionFieldNotClear

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/bug-894-component-visualSetting-default-mobileview-footer-overlapping-issue

abhijeetjawale1 committed:
minor change pushed for color varient

DeepashriDatar committed:
removed unnecessary code

hemalatagaikwad committed:
Merge pull request #910 from Wai-Technologies/user/pooja/bug-870-dynamic-entity-properties-badge-issue

Resolved Bug 870 :  Design System > Components > Dynamic Entity Properties> Default> Small, Large mobile overlap Parameter.

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/bug-894-component-visualSetting-default-mobileview-footer-overlapping-issue

rahulp34 committed:
commited changes- accordion bottom border hide issue resolved, and with icon stories updated as per figma wireframe

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/bug-898-component-ClaimType-Default-AfterSubmitionFieldNotClear

DeepashriDatar committed:
padding added for footer section

DeepashriDatar committed:
Merge branch 'users/deepashri/bug-894-component-visualSetting-default-mobileview-footer-overlapping-issue' of https://github.com/Wai-Technologies/raaghu-react into users/deepashri/bug-894-component-visualSetting-default-mobileview-footer-overlapping-issue

hemalatagaikwad committed:
Merge pull request #904 from Wai-Technologies/users/deepashri/bug-898-component-ClaimType-Default-AfterSubmitionFieldNotClear

Resolved Bug 898 : Design System > Components > Claim Type & Language > Default > After submission dropdown field does not clear

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/bug-894-component-visualSetting-default-mobileview-footer-overlapping-issue

hemalatagaikwad committed:
Merge pull request #899 from Wai-Technologies/users/deepashri/bug-894-component-visualSetting-default-mobileview-footer-overlapping-issue

Fixed bug 894 : Component > Visual Setting > default > mobile view (small & large) footer overlapping issue

hemalatagaikwad committed:
Merge branch 'development' into users/abhijeet-task-implement-different-types-of-avtaars

hemalatagaikwad committed:
Merge pull request #912 from Wai-Technologies/users/abhijeet-task-implement-different-types-of-avtaars

Completed Task 592 : Design System > Implement different types of Avtars

poojagurav21 committed:
committed changes for Button group alignment

poojagurav21 committed:
committed changes for label alignment

poojagurav21 committed:
committed changes for btn alignment

abhijeetjawale1 committed:
build issue resolved

abhijeetjawale1 committed:
Merge branch 'development' into users/abhijeet-build-issue-for-solution

hemalatagaikwad committed:
Merge pull request #924 from Wai-Technologies/users/abhijeet-build-issue-for-solution

Build Issues : Resolved Errors on Development Branch

poojagurav21 committed:
Merge branch 'development' into user/pooja/bug-889-secrets-btn-issue

abhijeetjawale1 committed:
Merge branch 'development' into users/abhijeet-task-enhancements-in-ratings

abhijeetjawale1 committed:
updated changes for the selection of the half star fill color

abhijeetjawale1 committed:
changes for the summary details

DeepashriDatar committed:
add changes for small mobile view

DeepashriDatar committed:
Merge branch 'development' into users/deepashri/task-590-enhancements-in-Alerts

poojagurav21 committed:
committed changs for resetting the required message after submitting

DeepashriDatar committed:
fixed alert element for small & large mobile views

poojagurav21 committed:
committed changes for Navtabs issue, version list clear after save and required message clear issue

poojagurav21 committed:
committed changes for removing stories

abhijeetjawale1 committed:
changes for the new element timer picker

hemalatagaikwad committed:
Merge pull request #915 from Wai-Technologies/users/abhijeet-task-enhancements-in-ratings

Completed Task 905 : Design System > Ratings > Enhancements in Ratings

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug-889-secrets-btn-issue

hemalatagaikwad committed:
Merge pull request #923 from Wai-Technologies/user/pooja/bug-889-secrets-btn-issue

Resolved Bug 889 : Design System > Components > Secrets > Default > Save cancel button alignment issue.

abhijeetjawale1 committed:
changes for the enhancement for the action column buttons

abhijeetjawale1 committed:
Merge branch 'development' into users/abhijeet-task-add-new-component-for-action

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/task-590-enhancements-in-Aerts

hemalatagaikwad committed:
Merge pull request #930 from Wai-Technologies/users/deepashri/task-590-enhancements-in-Aerts

Completed Task 590 : Design System > Enhancement > Implement different types of Alerts (For small & large mobile views)

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/bug-884-Components-PremiumSupport-Default-LargeMobileView-alignment-issue

hemalatagaikwad committed:
Merge pull request #928 from Wai-Technologies/users/deepashri/bug-884-Components-PremiumSupport-Default-LargeMobileView-alignment-issue

Resolved Bug 884 : Design System > Components > Premium Support> Default > Large mobile view card alignment is not display proper

hemalatagaikwad committed:
Merge branch 'development' into users/abhijeet-bug-summary-details-default-incorrect-text

hemalatagaikwad committed:
Merge pull request #927 from Wai-Technologies/users/abhijeet-bug-summary-details-default-incorrect-text

Resolved Bug 926 : Storybook > Components > Summary Details > Default > Incorrect Text.

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug-172-Acount-Ui-Issues

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug-660-label-alignment

abhijeetjawale1 committed:
changes for the button alignment

hemalatagaikwad committed:
Merge pull request #922 from Wai-Technologies/user/pooja/bug-660-label-alignment

Resolved Bug 660 & 844 : Storybook > Components > Change Password > Default > Error message alignment issue & Design System > Components > Register Member > Default > Display twice error message

abhijeetjawale1 committed:
changes for the go top text is not showing in dark mode

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/task-776-enhancements-in-toaster

poojagurav21 committed:
removed console.log

poojagurav21 committed:
Merge branch 'development' into user/pooja/bug-172-Acount-Ui-Issues

poojagurav21 committed:
committed changes for label alignment

rahulp34 committed:
commited changes- errors removed from build , some args are removed which is not required as per new wireframe

rahulp34 committed:
Merge remote-tracking branch 'origin/development' into users/deepashri/task-776-enhancements-in-toaster

rahulp34 committed:
Merge branch 'users/deepashri/task-776-enhancements-in-toaster' of https://github.com/Wai-Technologies/raaghu-react into users/deepashri/task-776-enhancements-in-toaster

poojagurav21 committed:
committed changes for color picker border

abhijeetjawale1 committed:
changes for the showing error while clicking on the outline or preview buttons

poojagurav21 committed:
committed changes for Feed UI issues

hemalatagaikwad committed:
Merge pull request #946 from Wai-Technologies/users/abhijeet-bug-change-theme-and-size-display-error-massage

Resolved Bug 807: Design System > Charts > Polar Area Chart > Change theme and size display error massage.

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug-172-Acount-Ui-Issues

hemalatagaikwad committed:
Merge pull request #932 from Wai-Technologies/user/pooja/bug-172-Acount-Ui-Issues

Resolved Bug 172 : Storybook > Component > Account > Default > Captcha > Issues on Account component Small, Large mobile view captcha not visible.

abhijeetjawale1 committed:
changes for the icons get hidden in small mobile view

abhijeetjawale1 committed:
changes for the dark and light theme stat is not displayed properly

poojagurav21 committed:
Changes for Introduction spelling mistake

hemalatagaikwad committed:
Merge pull request #951 from Wai-Technologies/user/pooja/bug-746-introduction-issue

Resolved Bug 746 : Design System > Version 8.3 > Scroll bar alignment issue.

nitinsonawane123 committed:
Resolved bug CenterIcon Icon is not change

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-804-centerIcon-name-is-not-change

hemalatagaikwad committed:
Merge branch 'development' into users/abhijeet-bug-dark-theme-go-top-button-is-no-visible

hemalatagaikwad committed:
Merge pull request #943 from Wai-Technologies/users/abhijeet-bug-dark-theme-go-top-button-is-no-visible

Resolved Bug 789 : Design System > Elements > Scrollspy > Dark theme Go Top button is not visible

hemalatagaikwad committed:
Merge branch 'development' into users/abhijeet-task-add-new-component-for-action

hemalatagaikwad committed:
Merge pull request #940 from Wai-Technologies/users/abhijeet-task-add-new-component-for-action

Completed Task 690: Design System > Components > Data Table > Add new component for Action

rahulp34 committed:
commited changes- sidebar responsiveness issue resolved for all the three stories

rahulp34 committed:
Merge remote-tracking branch 'origin/development' into users/rahul/task-820/Design-System-Elements-Side-navigation-enhancement

hemalatagaikwad committed:
Merge branch 'development' into users/abhijeet-bug-small-mobile-light-dark-theme-is-not-display-proper

hemalatagaikwad committed:
Merge pull request #950 from Wai-Technologies/users/abhijeet-bug-small-mobile-light-dark-theme-is-not-display-proper

Resolved Bug 792 : Design System > Elements > Stat > Advanced > Small mobile light & dark theme is not display proper

hemalatagaikwad committed:
Merge branch 'development' into users/abhijeet-bug-menus-directories-not-able-to-see-action-menus-small-view

hemalatagaikwad committed:
Merge pull request #948 from Wai-Technologies/users/abhijeet-bug-menus-directories-not-able-to-see-action-menus-small-view

Resolved Bug 811 :  Design System > Components > Menus Directories > Default > Small Mobile view > Not able to see action menus

hemalatagaikwad committed:
Merge branch 'development' into users/rahul/task-820/Design-System-Elements-Side-navigation-enhancement

rahulp34 committed:
Merge remote-tracking branch 'origin/development' into users/rahul/task-820/Design-System-Elements-Side-navigation-enhancement

hemalatagaikwad committed:
Merge branch 'development' into users/abhijeet-bug-button-alignment-issue

hemalatagaikwad committed:
Merge pull request #941 from Wai-Technologies/users/abhijeet-bug-button-alignment-issue

Resolved Bug 935 : Design System > Components > Add Member > Button alignment issue

hemalatagaikwad committed:
Merge branch 'development' into users/deepashri/task-776-enhancements-in-toaster

hemalatagaikwad committed:
Merge pull request #819 from Wai-Technologies/users/deepashri/task-776-enhancements-in-toaster

Completed Task 776 : Design System > Elements > Enhancements in Toast

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug-867-collaspible-container-icon-issue

hemalatagaikwad committed:
Merge pull request #913 from Wai-Technologies/user/pooja/bug-867-collaspible-container-icon-issue

Completed Bug 867 : Design System > Components > Collapsible Container > Default> UI issue and subscribe to icon color is light.

hemalatagaikwad committed:
Merge branch 'development' into users/rahul/Bug/94/Element-Accordion-Default-Dropdown-is-expected-to-close-after-clicking-on-it

hemalatagaikwad committed:
Merge pull request #917 from Wai-Technologies/users/rahul/Bug/94/Element-Accordion-Default-Dropdown-is-expected-to-close-after-clicking-on-it

Resolved Bug 94 : Element > Accordion > Default > Dropdown is expected to close after clicking on it

hemalatagaikwad committed:
Merge branch 'development' into user/rahul/Bug/866/Design-System-Components-ackground-Image-Default-Light-theme-selected-text-is-not-display-Proper

hemalatagaikwad committed:
Merge branch 'development' into user/prakash/enhancements-778-in-Spinner

hemalatagaikwad committed:
Merge pull request #838 from Wai-Technologies/user/prakash/enhancements-778-in-Spinner

Completed Task 778 : Design System > Elements > Enhancements in Spinner

hemalatagaikwad committed:
Merge branch 'development' into users/prakash/bug-893-tenant-dashboard-default-Small-mobile-view-Todo-list-issue

hemalatagaikwad committed:
Merge pull request #916 from Wai-Technologies/users/prakash/bug-893-tenant-dashboard-default-Small-mobile-view-Todo-list-issue

Resolved Bug 893 : Design System > Components > Tenant Dashboard > Default > Small mobile view Todo list issue

rahulp34 committed:
commited changes- updated alignment of profile icon and updated the view of search button in minimise state

rahulp34 committed:
Merge branch 'users/rahul/task-820/Design-System-Elements-Side-navigation-enhancement' of https://github.com/Wai-Technologies/raaghu-react into users/rahul/task-820/Design-System-Elements-Side-navigation-enhancement

abhijeetjawale1 committed:
changes for the accordian

nitinsonawane123 committed:
Resolved bug element alert text not showing proper

VenuKG committed:
Resolved bug- 958-Database Connection > Default > Connection Strings spacing issue.

nitinsonawane123 committed:
Resolved bug 936 introduction page incorrect text

hemalatagaikwad committed:
Merge branch 'development' into users/rahul/task-820/Design-System-Elements-Side-navigation-enhancement

hemalatagaikwad committed:
Merge pull request #961 from Wai-Technologies/users/rahul/task-820/Design-System-Elements-Side-navigation-enhancement

Resolved Bug 820 : Design System > Elements > Side navigation enhancements

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug-882-btn-alignment

hemalatagaikwad committed:
Merge pull request #921 from Wai-Technologies/user/pooja/bug-882-btn-alignment

Resolved Bug 882 : Design System > Components > File Management Tree > Default > Small large view does not display proper.

hemalatagaikwad committed:
Merge branch 'development' into users/nitin/bug-936-introduction-page-incorrect-text

hemalatagaikwad committed:
Merge pull request #970 from Wai-Technologies/users/nitin/bug-936-introduction-page-incorrect-text

Resolved Bug 936 : Design System > Introduction > Incorrect text

hemalatagaikwad committed:
Merge branch 'development' into users/nitin/bug-963-element-alert-incorrect-text

hemalatagaikwad committed:
Merge pull request #966 from Wai-Technologies/users/nitin/bug-963-element-alert-incorrect-text

Resolved Bug 963 : Design System >Elements > Alert > Default > Incorrect text

hemalatagaikwad committed:
Merge branch 'development' into users/nitin/bug-804-centerIcon-name-is-not-change

hemalatagaikwad committed:
Merge pull request #953 from Wai-Technologies/users/nitin/bug-804-centerIcon-name-is-not-change

Resolved Bug 804 : Design System > Version 8.3 > Charts > Boolean Chart > CenterIcon Icon is not change. #804

abhijeetjawale1 committed:
changes for the view border bottom for accordian

abhijeetjawale1 committed:
Merge branch 'development' into users/abhijeet-bug-747-accordion-border-is-not-visible

hemalatagaikwad committed:
Merge branch 'development' into users/abhijeet-task-create-new-element-time-picker

abhijeetjawale1 committed:
changes for renamed props name

hemalatagaikwad committed:
Merge pull request #974 from Wai-Technologies/users/abhijeet-bug-747-accordion-border-is-not-visible

Resolved Bug 747 : Design System > Elements > Accordion > Border is not visible proper.

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug-732-RegisterMember-label-issue

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug-787-element-feed-UI-issues

hemalatagaikwad committed:
Merge pull request #947 from Wai-Technologies/user/pooja/bug-787-element-feed-UI-issues

Resolved Bug 787: Design System > Version 8.3 > Elements > Feed > For Small & large mobile view text alignment is not proper and duplicate text is displayed

poojagurav21 committed:
changes for checkbox alignment

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug-732-RegisterMember-label-issue

hemalatagaikwad committed:
Merge pull request #944 from Wai-Technologies/user/pooja/bug-732-RegisterMember-label-issue

Resolved Bug 732 : Storybook > Component > Register Member >Label alignment issue.

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug-764-element-bigNumber-dark-theme-issue

hemalatagaikwad committed:
Merge pull request #945 from Wai-Technologies/user/pooja/bug-764-element-bigNumber-dark-theme-issue

Resolved Bug 764 : Design System > Version 8.3.3 >Elements> Big Number > Default > Dark theme number is not display proper

hemalatagaikwad committed:
Merge branch 'development' into users/abhijeet-task-create-new-element-time-picker

hemalatagaikwad committed:
Merge pull request #934 from Wai-Technologies/users/abhijeet-task-create-new-element-time-picker

Completed Task 900 : Design System > Element > Create new element Time Picker

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/enhancement-788-text-editor-story-issue

hemalatagaikwad committed:
Merge pull request #933 from Wai-Technologies/user/pooja/enhancement-788-text-editor-story-issue

Completed Task 788 : Design System > Elements > Enhancements in Text Editor

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug-871-aftersubmitting-required-msg-not-clear

hemalatagaikwad committed:
Merge pull request #929 from Wai-Technologies/user/pooja/bug-871-aftersubmitting-required-msg-not-clear

Resolved Bug 871 : Design System > Components > Required message is not displaying, & validation message alignment issue,

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug-883-component-invoice-detail-receipt-text-issue

hemalatagaikwad committed:
Merge pull request #909 from Wai-Technologies/user/pooja/bug-883-component-invoice-detail-receipt-text-issue

Resolved Bug 883: Design System > Components > Invoice Detail Receipt> Default >Dark theme text is not display proper

hemalatagaikwad committed:
Merge branch 'development' into users/venu/bug-958-database-connection-strings-spacing-issue

hemalatagaikwad committed:
Merge pull request #967 from Wai-Technologies/users/venu/bug-958-database-connection-strings-spacing-issue

Resolved Bug 958 : Design System > Database Connection > Default > Connection Strings spacing issue.

hemalatagaikwad committed:
Merge branch 'development' into user/rahul/Bug/866/Design-System-Components-ackground-Image-Default-Light-theme-selected-text-is-not-display-Proper

hemalatagaikwad committed:
Merge pull request #914 from Wai-Technologies/user/rahul/Bug/866/Design-System-Components-ackground-Image-Default-Light-theme-selected-text-is-not-display-Proper

Resolved Bug 866 : Design System > Components > Background Image > Default >Light theme selected text is not display Proper

hemalatagaikwad committed:
Merge branch 'development' into users/hemalata/task-move-dev-changes-to-stage-branch
